frontend/src/components/Psychologist/Profile/EditProfile.jsx

```diff
@@ -1,13 +1,14 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 
 import Ariane from 'components/Ariane/Ariane';
 import GlobalNotification from 'components/Notification/GlobalNotification';
 import Input from 'components/Form/Input';
 
-import { REGIONS, DEPARTEMENTS } from 'services/geo';
+import DEPARTEMENTS from 'services/departments';
```

Comment:
We could use `departementsToRegion.keys()` from `geo.ts` to avoid duplicating departments data.